pandas/_libs/tslibs/np_datetime.pyx

```diff
@@ -1,12 +1,15 @@
 from cpython.object cimport Py_EQ, Py_NE, Py_GE, Py_GT, Py_LT, Py_LE
 
-from cpython.datetime cimport (datetime, date,
-                               PyDateTime_IMPORT,
-                               PyDateTime_GET_YEAR, PyDateTime_GET_MONTH,
-                               PyDateTime_GET_DAY, PyDateTime_DATE_GET_HOUR,
```

Comment:
i find the year-month-day-hour-minute-second ordering clearer than the alphabetical ordering.  its not worth bikeshedding, which is why id discourage these for the most part

also makes it harder to see what the actual removed imports are (datetime and date, right?).  That part im on board with.

Reply:
> i find the year-month-day-hour-minute-second ordering clearer than the alphabetical ordering. its not worth bikeshedding, which is why id discourage these for the most part

Agreed, I have no idea how to enforce it with ```isort```

> also makes it harder to see what the actual removed imports are (datetime and date, right?). That part im on board with.

Correct, that's why I did it in two commits and not in one, as you an see after the removal of the unused imports what is left looks pretty ugly, so I figured I run ```isort``` on that specific import block.